### PR TITLE
data: shaileshthakur1

### PR DESCRIPTION
**Pull request by shaileshthakur1**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/shaileshthakur1.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Shailesh Thakur",
+  "displayStatsPublic": true,
```
Comment (sital002) [inline data/shaileshthakur1.json:3]:
```suggestion
```
  ```displayStatsPublic``` is not required.

File: data/shaileshthakur1.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "type": "personal",
```
Comment (sital002) [inline data/shaileshthakur1.json:4]:
```suggestion
```
```type``` isnot required

File: data/shaileshthakur1.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "bio": "Currently a 2nd year Undergrad,interested in project and research internship from abroad",
+  "avatar": "https://github.com/shaileshthakur1.png"
```
Comment (sital002) [inline data/shaileshthakur1.json:6]:
```suggestion
  "avatar": "https://github.com/shaileshthakur1.png"
```
```avatar``` property isnot required

**Review by sital002 — CHANGES_REQUESTED** [review]
Please make the necessary changes to get your profile merged.

**Comment by shaileshthakur1** [comment]
hey @sital002 , I've updated it all. Please review now.

**Review by sital002 — APPROVED** [review]
Looks good 👍

**Pull request merged**